cmd/minishift/cmd/root.go

```diff
@@ -193,6 +185,11 @@ func init() {
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+	profile := initializeProfile()
```

Comment:
creates a local variable that overrides https://github.com/minishift/minishift/pull/1380/files?diff=unified#diff-caeade0d74523273373d2689861f618eR54 :-s